components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/modules/policy.js

```diff
@@ -136,12 +136,12 @@ policyModule = function () {
         var mqttBrokerPort = mqttEndPointDeviceConfig.getPort();
         var mqttQueueEndpoint = mqttBrokerURL + ":" + mqttBrokerPort;
 
-        var mqttsenderClass = Packages.org.wso2.carbon.device.mgt.iot.mqtt.PolicyPush;
-        var mqttsender = new mqttsenderClass();
+        var mqttSenderClass = Packages.org.wso2.carbon.device.mgt.iot.mqtt.PolicyPush;
+        var mqttSender = new mqttSenderClass();
```

Comment:
Ack, Fixed in commit https://github.com/wso2/carbon-device-mgt/pull/304/commits/d2daa0507eb9eb1b5d149fde50e305634e0f6b51